.pre-commit-config.yaml

```diff
@@ -0,0 +1,18 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.6.0
+    hooks:
+      - id: check-added-large-files
+      - id: check-merge-conflict
+      - id: check-yaml
+      - id: debug-statements
+      - id: detect-aws-credentials
+        args:
+          - --allow-missing-credentials
+      - id: end-of-file-fixer
+      - id: no-commit-to-branch
```

Comment:
I think I might prefer just removing this one and running on main

Reply:
Maybe, but some people _have_ accidentally committed to and pushed to `main` here (won't name names 🙊 ) so for now I prefer to keep it.

Reply:
I think in the monorepo we just do `pre-commit run --all-files` and for some reason that sidesteps this check. unclear why